citations/stat.js

```diff
@@ -29,13 +29,12 @@ module.exports = {
   ],
 
   links: function(cite) {
-    // GPO only has volumes starting with 65.
-    if (cite.volume < 65) return { };
+    var links = { };
 
-    var usgpo_url = "http://api.fdsys.gov/link?collection=statute&volume=" + cite.volume + "&page=" + cite.page;
-
-    return {
-      usgpo: {
+    // GPO
+    if (cite.volume >= 65 && cite.volume <= 125) {
```

Comment:
So I'll take that as a strong feeling. :) From my vantage point, giving out broken links on the current end of the spectrum -- especially broken links which will eventually become unbroken -- seems better than setting up the library to be perennially behind in giving out working links. 

But you're the main user of the links and I'm happy to give it a shot and change the behavior if it ends up going poorly!